source/making_payments/index.html.md.erb

```diff
@@ -5,120 +5,139 @@ weight: 70
-section of ``_links`` in the JSON body (the same URL is shown in both places).
-This URL contains the GOV.UK Pay ``payment_id`` which identifies the
-payment. An authenticated GET request to the URL will return information about
-the payment and its status.
+You will need to store the URL from the `Location` response header, or from
+the `self` section of `_links` in the JSON body.  This URL contains the GOV.UK
+Pay `payment_id`, which identifies the payment. You can make a GET request to
+this URL to return information about the payment and its status.
 
-You can read more about this in the [Reporting](/reporting) section of the
+You can read more in the [__Reporting__](/reporting) section of this
 documentation.
 
 An example URL:
 `https://publicapi.payments.service.gov.uk/v1/payments/{paymentId}`
 
->It is important that you do not expose the URL with the ``payment_id``
-publicly, for example as a URL parameter or in an insecure cookie. You should
-store it as a secure cookie or in a database.
+<style>
+.govuk-warning-text{font-family:nta,Arial,sans-serif;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;font-weight:400;font-size:16px;font-size:1rem;line-height:1.25;color:#0b0c0c;position:relative;margin-bottom:20px;padding:10px 0}@media print{.govuk-warning-text{font-family:sans-serif}}@media (min-width:40.0625em){.govuk-warning-text{font-size:19px;font-size:1.1875rem;line-height:1.31579}}@media print{.govuk-warning-text{font-size:14pt;line-height:1.15;color:#000}}@media (min-width:40.0625em){.govuk-warning-text{margin-bottom:30px}}.govuk-warning-text__assistive{position:absolute!important;width:1px!important;height:1px!important;margin:-1px!important;padding:0!important;overflow:hidden!important;clip:rect(0 0 0 0)!important;-webkit-clip-path:inset(50%)!important;clip-path:inset(50%)!important;border:0!important;white-space:nowrap!important}.govuk-warning-text__icon{font-family:nta,Arial,sans-serif;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;font-weight:700;display:inline-block;position:absolute;top:50%;left:0;min-width:32px;min-height:29px;margin-top:-20px;padding-top:3px;border:3px solid #0b0c0c;border-radius:50%;color:#fff;background:#0b0c0c;font-size:1.6em;line-height:29px;text-align:center;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none}@media print{.govuk-warning-text__icon{font-family:sans-serif}}.govuk-warning-text__text{display:block;margin-left:-15px;padding-left:65px}
+</style>
 
-You will receive the ``next_url``  to which you should direct the user to
+<br>
+
+<div class="govuk-warning-text">
+  <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+  <strong class="govuk-warning-text__text">
+    <span class="govuk-warning-text__assistive">Warning</span>
+You must not expose the URL with the ``payment_id`` publicly, for example as a URL parameter or in an insecure cookie. You should store it as a secure cookie or in a database.
```

Comment:
This doesn’t quite seem to fit. The `next_url` has a token in it but it’s not the `payment_id`. Bot the `next_url` and the `payment_id` should be kept secure.